src/Components/Home/ListItem.js

```diff
@@ -10,16 +10,15 @@ function ListItem(prop) {
              <Row>
                 <Col>
                     {daysLeft == 0 ?  
-                        <Button variant="outline-dark" className="px-3"><i className="fas fa-shopping-cart"></i> Buy</Button>: 
+                        <Button variant="danger" className="px-3"><i className="fas fa-minus-circle"></i> Forgot</Button>: 
                         <span><i className="far fa-clock"></i> {daysLeft} days left</span>}
                 </Col>
                 <Col className="text-right">
-                    <Button variant="outline-dark"><i className="fas fa-pen" ></i></Button>
-                    <Button variant="outline-dark" style={{paddingLeft:'0.55em', paddingRight:'0.2em'}}>
+                    <Button variant="warning"><i className="fas fa-pen" ></i></Button>
```

Comment:
In my opinion, we have now too many buttons grabbing the user's attention. If all buttons have strong colors they will seem to have the same importance to the user. This button doesn't need to stand out.
```suggestion
                    <Button variant="outline-secondary"><i className="fas fa-pen" ></i></Button>
```

Reply:
have implemented